x-pack/plugins/translations/translations/fr-FR.json

```diff
@@ -26886,11 +26886,7 @@
     "xpack.observability.slo.slo.deleteConfirmationModal.cancelButtonLabel": "Annuler",
     "xpack.observability.slo.slo.deleteConfirmationModal.title": "Voulez-vous vraiment continuer ?",
     "xpack.observability.slo.slo.item.actions.clone": "Cloner",
-    "xpack.observability.slo.slo.item.actions.createRule": "Créer une règle d'alerte",
```

Comment:
If you rename these to the updated translation key it saves the translation agency from doing another pass of translations:

`"xpack.observability.slo.item.actions.createRule": "Créer une règle d'alerte",`